extensions/wikia/JsonFormat/util/DomHelper.php

```diff
@@ -126,4 +126,8 @@ public static function getTextValue( DomNode $node, $allow = ['#text','a','b','i
 		return trim( $text );
 	}
 
+	public static function cleanDescendantHeaderInternalCache() {
+		self::$hasDescendantHeaderInternalCache = [];
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)